0-bootstrap/main.tf

```diff
@@ -77,20 +87,7 @@ module "seed_bootstrap" {
     "billingbudgets.googleapis.com"
   ]
 
-  sa_org_iam_permissions = [
-    "roles/accesscontextmanager.policyAdmin",
-    "roles/billing.user",
-    "roles/compute.networkAdmin",
-    "roles/compute.xpnAdmin",
-    "roles/iam.securityAdmin",
-    "roles/iam.serviceAccountAdmin",
-    "roles/logging.configWriter",
-    "roles/orgpolicy.policyAdmin",
-    "roles/resourcemanager.projectCreator",
-    "roles/resourcemanager.folderAdmin",
-    "roles/securitycenter.notificationConfigEditor",
-    "roles/resourcemanager.organizationViewer"
-  ]
+  sa_org_iam_permissions = []
```

Comment:
we need it because the default value is not empty:

https://github.com/terraform-google-modules/terraform-google-bootstrap/blob/a300b9c27796200f2a565173484eb997d8e296a8/variables.tf#L98-L108
```hcl
  default = [
    "roles/billing.user",
    "roles/compute.networkAdmin",
    "roles/compute.xpnAdmin",
    "roles/iam.securityAdmin",
    "roles/iam.serviceAccountAdmin",
    "roles/logging.configWriter",
    "roles/orgpolicy.policyAdmin",
    "roles/resourcemanager.folderAdmin",
    "roles/resourcemanager.organizationViewer",
  ]
```